app/src/main/java/com/dilaefendioglu/dogdiscovery/data/DogFreeResponse.kt

```diff
@@ -0,0 +1,17 @@
+package com.dilaefendioglu.dogdiscovery.data
+
+import java.io.Serializable
+
+data class DogFreeResponse(
+    val id: Int,
+    val name: String,
+    val breed_group: String,
```

Comment:
```suggestion
        @SerializedName("breed_group") val group: String,
```
underscore yerine camelCase kullanalım.